src/components/form/GitForm.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <q-form
+      @submit="onSubmit"
+      class="q-gutter-md">
```

Comment:
Set a minimum width for the modal so that it is not resized when the input is incorrect.

Reply:
Done